sendrecv/gst-rust/src/main.rs

```diff
@@ -0,0 +1,480 @@
+extern crate clap;
+extern crate glib;
+extern crate gstreamer as gst;
+extern crate gstreamer_sdp;
```

Comment:
I'd recommend importing this as `gst_sdp` (and below `gst_webrtc`) for consistency and less typing